### ISSUE TITLE
Compatiblity issue AEX2 Rpc

### ISSUE THREAD

**Issue by mradkov** [waellet/waellet#172]
There is a problem with aex-2 rpc connection (#146 , https://github.com/aeternity/aepp-waellet/pull/98) with browsers different than Chrome (e.g. brave, firefox)

Brave error log:
```
persistState.js:19 Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at b (https://base.aepps.com/js/app.b31db344.js:1:50783)
    at https://base.aepps.com/js/app.b31db344.js:1:51053
    at https://base.aepps.com/js/chunk-vendors.bf94ccbc.js:13:3560
    at Array.forEach (<anonymous>)
    at new y (https://base.aepps.com/js/chunk-vendors.bf94ccbc.js:13:3533)
    at Object.<anonymous> (https://base.aepps.com/js/app.b31db344.js:1:17201)
    at Object.4360 (https://base.aepps.com/js/app.b31db344.js:1:18718)
    at s (https://base.aepps.com/js/app.b31db344.js:1:1410)
    at Module.<anonymous> (https://base.aepps.com/js/app.b31db344.js:1:18871)
    at Module.56d7 (https://base.aepps.com/js/app.b31db344.js:1:19103)
    at s (https://base.aepps.com/js/app.b31db344.js:1:1410)
    at Object.0 (https://base.aepps.com/js/app.b31db344.js:1:4913)
    at s (https://base.aepps.com/js/app.b31db344.js:1:1410)
    at n (https://base.aepps.com/js/app.b31db344.js:1:431)
    at https://base.aepps.com/js/app.b31db344.js:1:4878
    at https://base.aepps.com/js/app.b31db344.js:1:4883
```

Firefox log:
```
ReferenceError: chrome is not definedbundle.js:43:57267
```

cc @petbaik

**Comment by petbaik** [comment]
> There is a problem with aex-2 rpc connection (#146 , [aeternity#98](https://github.com/aeternity/aepp-waellet/pull/98)) with browsers different than Chrome (e.g. brave, firefox)
> 
> Brave error log:
> 
> ```
> persistState.js:19 Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
>     at b (https://base.aepps.com/js/app.b31db344.js:1:50783)
>     at https://base.aepps.com/js/app.b31db344.js:1:51053
>     at https://base.aepps.com/js/chunk-vendors.bf94ccbc.js:13:3560
>     at Array.forEach (<anonymous>)
>     at new y (https://base.aepps.com/js/chunk-vendors.bf94ccbc.js:13:3533)
>     at Object.<anonymous> (https://base.aepps.com/js/app.b31db344.js:1:17201)
>     at Object.4360 (https://base.aepps.com/js/app.b31db344.js:1:18718)
>     at s (https://base.aepps.com/js/app.b31db344.js:1:1410)
>     at Module.<anonymous> (https://base.aepps.com/js/app.b31db344.js:1:18871)
>     at Module.56d7 (https://base.aepps.com/js/app.b31db344.js:1:19103)
>     at s (https://base.aepps.com/js/app.b31db344.js:1:1410)
>     at Object.0 (https://base.aepps.com/js/app.b31db344.js:1:4913)
>     at s (https://base.aepps.com/js/app.b31db344.js:1:1410)
>     at n (https://base.aepps.com/js/app.b31db344.js:1:431)
>     at https://base.aepps.com/js/app.b31db344.js:1:4878
>     at https://base.aepps.com/js/app.b31db344.js:1:4883
> ```
> 
> Firefox log:
> 
> ```
> ReferenceError: chrome is not definedbundle.js:43:57267
> ```
> 
> cc @petbaik

The brave error log is connected to injected iframe of base.aepps in the example aepp we use.

The firefox error is connected to https://github.com/aeternity/aepp-sdk-js/blob/feature/aex-2/es/utils/aepp-wallet-communication/helpers.js#L3. When you call chrome in the if condition like this in firefox - **error chrome is not defined** is thrown. I think the check should be `typeof chrome != "undefined"`

**Comment by nduchak** [comment]
Hey, 
It's fixed in this PR: https://github.com/aeternity/aepp-sdk-js/pull/853/files
Please reply if it's fixed for you

**Comment by mradkov** [comment]
@petbaik Is this resolved already with 7.1.1?

**Comment by petbaik** [comment]
> Is this resolved already with 7.1.1?

resolved
